Tests/FunctionalTests/TestRig/Sources/FTRRotatedViewsViewController.m

```diff
@@ -20,6 +20,14 @@ @implementation FTRRotatedViewsViewController
 
 #pragma mark - Event handling
 
+- (void)motionEnded:(UIEventSubtype)motion withEvent:(UIEvent *)event
+{
+	if(event.subtype == UIEventSubtypeMotionShake)
+	{
```

Comment:
Hey, formatting is off here - 

http://google.github.io/styleguide/objcguide.html

Reply:
Fixed